dgg-utilities.user.js

```diff
@@ -1514,1215 +1431,988 @@ function injectScript() {
-                                    if (splitHeader[0] == "etag") {
-                                        phrasesEtag = splitHeader[1].substring(
-                                            3,
-                                            splitHeader[1].length - 1
-                                        );
-                                    }
-                                });
-                            phrases = [];
-                            data.forEach((entry) => {
-                                cleanPhrase = entry.phrase.trim().toLowerCase();
-                                if (
-                                    /^\/.*\/$/.test(cleanPhrase) &&
-                                    cleanPhrase.length > 2
-                                ) {
-                                    const regexString = cleanPhrase.slice(
-                                        1,
-                                        cleanPhrase.length - 1
-                                    );
-                                    const regex = new RegExp(regexString, "i");
-                                    phrases.push(regex);
-                                } else {
-                                    phrases.push(entry.phrase);
-                                }
-                            });
-                        } else if (response.status != 304) {
-                            console.error(
-                                `[ERROR] [dgg-utils] couldn't get the phrase data - URL: "https://mitchdev.net/api/dgg/list", HTTP status code: ${response.status} - ${response.statusText}`
-                            );
-                        }
-                    },
-                    onerror: () => {
-                        console.error(
-                            `[ERROR] [dgg-utils] couldn't get the phrase data - URL: "https://mitchdev.net/api/dgg/list", HTTP error`
-                        );
-                    },
-                    ontimeout: () => {
-                        console.error(
-                            `[ERROR] [dgg-utils] couldn't get the phrase data - URL: "https://mitchdev.net/api/dgg/list", HTTP timeout`
-                        );
-                    },
-                });
-            },
-        });
-    }
-
-    getPhrases();
-
-    function textScanner(event) {
-        // ensure we dont fire on random empty keypresses
-        if (
-            !(
-                event.code == "ControlLeft" ||
-                event.code == "ControlRight" ||
-                event.code == "AltLeft" ||
-                event.code == "AltRight" ||
-                event.code == "ShiftLeft" ||
-                event.code == "ShiftRight" ||
-                event.code == "MetaLeft" ||
-                event.code == "MetaRight"
-            )
-        ) {
-            let text = textarea.value.toLowerCase();
-            let resultCustom;
-            let resultCustomSoft;
```

Comment:
Seems like you removed this line (```let resultCustomSoft;```) on accident while cleaning up, gotta add it back (leads to errors if array is empty) NODDERS

Reply:
To remove the auto formatting I took your base version and cv pasted my stuff back into it and I guess I forgor that line nathanDerp